props/saber_fett263_buttons.h

```diff
@@ -45,15 +85,48 @@ SaberFett263Buttons() : PropBase() {}
   bool swinging_ = false;
   void Loop() override {
     PropBase::Loop();
-    if (!swinging_ && fusor.swing_speed() > 250) {
-      swinging_ = true;
-      Event(BUTTON_NONE, EVENT_SWING);
-    }
-    if (swinging_ && fusor.swing_speed() < 100) {
-      swinging_ = false;
+    // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+    if (!SaberBase::IsOn()) {
```

Comment:
In general, it's easier to read code if conditions are not negated.
So, I would recommend changing this to if (!SaberBase::IsOn()) { and then switch the two code blocks below around.

Reply:
Sorry, I don't understand what am I changing?

Reply:
I would recommend changing it from:

  if (!SaberBase::IsOn()) {
      CODE BLOCK A
  } else {
      CODE BLOCK B
  }

to:
  if (SaberBase::IsOn()) {
     CODE BLOCK B
  } else {
     CODE BLOCK A
  }

That makes it easier to follow the code because you don't need to think "else not on".

Reply:
ok, I see I have changed.  The indents are still different I look through everything in github.com and it's lined up as you explained so not sure what's going on there